lib/node-package-manager.ts

```diff
@@ -92,7 +92,13 @@ export class NodePackageManager implements INodePackageManager {
 	public view(packageName: string, config: any): IFuture<any> {
 		return (() => {
 			let flags = this.getFlagsString(config, false);
-			let viewResult = this.$childProcess.exec(`npm view ${packageName} ${flags}`).wait();
+			let viewResult: any;
+			try {
+				viewResult = this.$childProcess.exec(`npm view ${packageName} ${flags}`).wait();
+			} catch(e) {
+				this.$logger.out(e);
```

Comment:
What's the reason to catch the error, print it and throw it again? In case `npm view` is used in a command, CLI's commandsService will print the error when it's thrown.